app/scripts/controllers/overview.js

```diff
@@ -1101,6 +1101,7 @@ function OverviewController($scope,
     // - API objects by binding name
     state.bindingsByApplicationUID = {};
     state.applicationsByBinding = {};
+    state.validBindingsByApplicationUID = {};
```

Comment:
I don't have a great suggestion :(

 `valid` sort of implies ready to me... `deleteableBindingsByApplicationUID` too on the nose?